lib/worker/batcher.ex

```diff
@@ -538,13 +539,9 @@ defmodule BorsNG.Worker.Batcher do
     |> Enum.map(fn {context, _} -> context end)
     |> MapSet.new()
     |> MapSet.disjoint?(MapSet.new(toml.pr_status))
-    passed_review = if is_nil(toml.required_approvals) do
-      :sufficient
-    else
-      repo_conn
-      |> GitHub.get_reviews!(patch.pr_xref)
-      |> are_reviews_passing(toml.required_approvals)
-    end
+    passed_review = repo_conn
+    |> GitHub.get_reviews!(patch.pr_xref)
+    |> are_reviews_passing(toml)
```

Comment:
Note: I wrote this way to keep with the style, but personally I think that it's more readable when we indent it in the next block:

```
review_status = 
  repo_conn
  |> GitHub.get_reviews!(patch.pr_xref)
  |> are_reviews_passing(toml)
```